client/src/components/ProfileEdit/data.js

```diff
@@ -0,0 +1,34 @@
+const range = (from, to) => { 
+	const arr = [];
+	for (let i = from; i <= to; i++) { 
+		arr[arr.length] = i;
+	}
+	return arr;
+}
+
+const year = new Date().getFullYear();
+
+export const genders = [
+	'Male',
+	'Female',
+	'Non-Binary',
+	'Prefer Not to Say'
+];
+
```

Comment:
It feels like the below can be avoided if you instead used https://material-ui.com/components/pickers/#datepickers